docs/oscar_references.bib

```diff
@@ -1385,6 +1385,19 @@ @PhDThesis{Lev05
   school        = {Technische Universität Kaiserslautern}
 }
 
+@Book{Liu06,
+  author        = {Liu, Qing},
+  title         = {Algebraic geometry and arithmetic curves. {Transl}. by {Reinie} {Ern{\'e}}},
```

Comment:
Our bibtex implementation does not like braces and escapes in the title. Please use unicode instead. Casing is preserved without anything to do.